test/unit/test_run_reports.py

```diff
@@ -7,22 +7,9 @@
 reports = [ReportConfig("r1", None, ["r1@example.com"], "Subj: r1"),
            ReportConfig("r2", {"p1": "v1"}, ["r2@example.com"], "Subj: r2")]
 
-send_email_call_r1 = call(
```

Comment:
Similarly here tests are basically unchanged but as email sending is no longer baked into the `run_reports` method tests use an arbitrary success callback instead.